LICENSE.md

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2022 EddieHubCommunity
```

Comment:
Question, possibly for @eddiejaoude because I'm not seeing a straight answer on the internet.  Since tomorrow (or already for some folks) it's 2023 should we put 2023 on this? Or a range 2022 -2023?  Or is it okay to simply have the year the software was created?

Reply:
Did some more looking, i think this is okay.  Can always update later if I was wrong.  Thank you!

Reply:
We have this on LinkFree but we probably need to review all of our projects, as you mentioned we can always update later 
https://github.com/EddieHubCommunity/LinkFree/blob/main/LICENSE